src/nodes/gr/gr.h

```diff
@@ -47,6 +47,12 @@ class GRNode : public Node
     private:
     // Target configurations
     std::vector<Configuration *> targetConfigurations_;
+    // Original g(r)
+    std::optional<PartialSet> originalgr_;
+    // Unweighted g(r)
+    std::optional<PartialSet> unweightedgr_;
+    // Summed unweighted g(r)
+    std::optional<PartialSet> summedunweightedgr_;
```

Comment:
Mr Picky Names here again:
```suggestion
    std::optional<PartialSet> originalGR_;
    // Unweighted g(r)
    std::optional<PartialSet> unweightedGR_;
    // Summed unweighted g(r)
    std::optional<PartialSet> summedUnweightedGR_;
```